moveit_ros/planning/planning_request_adapter_plugins/src/fix_workspace_bounds.cpp

```diff
@@ -65,9 +65,9 @@ class FixWorkspaceBounds : public planning_request_adapter::PlanningRequestAdapt
     return "Fix Workspace Bounds";
   }
 
-  bool adaptAndPlan(const PlannerFn& planner, const planning_scene::PlanningSceneConstPtr& planning_scene,
-                    const planning_interface::MotionPlanRequest& req,
-                    planning_interface::MotionPlanResponse& res) const override
+  bool adapt(const planning_scene::PlanningSceneConstPtr& /*planning_scene*/,
+             planning_interface::MotionPlanRequest& req,
+             const planning_interface::MotionPlanResponse& /*res*/) const override
```

Comment:
This one seems like it wouldn't need the planner, since it is just mutating the workspace bounds. ✅